pkg/kubectl/cmd/util/printing.go

```diff
@@ -28,12 +29,12 @@ import (
 
 // AddPrinterFlags adds printing related flags to a command (e.g. output format, no headers, template path)
 func AddPrinterFlags(cmd *cobra.Command) {
-	cmd.Flags().StringP("output", "o", "", "Output format. One of: json|yaml|template|templatefile|wide|jsonpath.")
+	cmd.Flags().StringP("output", "o", "", "Output format. One of: json|yaml|wide|gotemplate=...|gotemplate-file=...|jsonpath=...|jsonpath-file=... See golang template [http://golang.org/pkg/text/template/#pkg-overview] and jsonpath template [http://releases.k8s.io/HEAD/docs/user-guide/jsonpath.md].")
```

Comment:
@jlowdermilk: @thockin indeed suggested changing to `-o gotemplate=...`. One flag.